### ISSUE TITLE
1.3.4 is borked with Browserify

### ISSUE THREAD

**Issue by timothyallan** [casesandberg/react-color#78]
I've made sure I got the 1.3.4 version with the reverted export fix.

Just doing a simple 
`const ColorPicker = require('react-color');`

and my usual

`<ColorPicker type="compact"/>`

Which has been working for months. 

Chrome is now spitting out:

Warning: React.createElement: type should not be null, undefined, boolean, or number. It should be a string (for DOM elements) or a ReactClass (for composite components).

Uncaught Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

**Comment by manugb** [comment]
Yep, its happening to me as well. I upgraded to 1.3.4 and the React-Color import stopped working. Using webpack.

Using react-color version 1.3.0 seems to be working.

@timothyallan if you want the last version using this should work:

`<ColorPicker.default type='compact'>`

just add the `.default` to ColorPicker

**Comment by casesandberg** [comment]
Sorry guys, I broke something somewhere and there isnt test coverage for it. I am going to get this fixed today and make sure there is coverage for it in the future

**Comment by timothyallan** [comment]
Cool, I'll keep an eye out for 1.3.5 ;)

In the meantime thanks @manugb, that fixed it

**Comment by frank-weindel** [comment]
In webpack I get these errors with 1.3.4:

```
ERROR in ./~/react-color/modules/react-material-design/lib/components/Tabs.js
Module not found: Error: Cannot resolve module 'lodash/lang/isString' in /Users/frank-weindel/Repo/vi/vi-admin/node_modules/react-color/modules/react-material-design/lib/components
 @ ./~/react-color/modules/react-material-design/lib/components/Tabs.js 11:16-47

ERROR in ./~/react-color/modules/react-material-design/lib/components/Link.js
Module not found: Error: Cannot resolve module 'lodash/lang/isString' in /Users/frank-weindel/Repo/vi/vi-admin/node_modules/react-color/modules/react-material-design/lib/components
 @ ./~/react-color/modules/react-material-design/lib/components/Link.js 9:16-47
```

Seems like certain lodash modules aren't being installed!

**Comment by chrishouse** [comment]
I'm also receiving an error stating that the module 'lodash/lang/isString' cannot be found.

**Comment by casesandberg** [comment]
This should be fixed in `react-color@1.3.6` please let me know if you are still having issues!
